README.md

```diff
@@ -207,6 +207,46 @@ There is available REST API for AMOC where we can:
 * check status of nodes in AMOC cluster
 
 API is described [here](REST_API_DOCS.md). You can also get an access to REST API by running AMOC and go to `(address:port)/api-docs`.
-### Docker
 
-TODO
+## Docker
+
+To build a Docker image with Amoc, run the following command from the root of
+the repository:
+```
+docker build -f Dockerfile -t image .
```

Comment:
I think it'd better to put more meaningful tag in the example `image` can be applied to anything and people tend to just copy and paste commands when try things.